NAMESPACE

```diff
@@ -86,6 +105,12 @@ export(ft_quantile_discretizer)
 export(ft_sql_transformer)
 export(ft_string_indexer)
 export(ft_vector_assembler)
+export(hive_context)
```

Comment:
I wonder if the methods here should become `spark_hive_context`, `spark_invoke`, and so on?